nubis/files/salesforce/sfdc_populate_sf_summary_table.py

```diff
@@ -0,0 +1,44 @@
+#!/bin/env python
+
+import pyodbc
+
+cnxn = pyodbc.connect("DSN=vertica")
```

Comment:
pyodbc uses auto-commit by default, annoyingly.
```
pyodbc.connect("DSN=vertica", autocommit=False)
```
Otherwise, you are not as atomic as you think you are supposed to be.